Atomic/util.py

```diff
@@ -33,6 +33,7 @@
 BWRAP_OCI_PATH = "/usr/bin/bwrap-oci"
 RUNC_PATH = "/bin/runc"
 SKOPEO_PATH = "/usr/bin/skopeo"
+POLICY_FILENAME = "/etc/containers/policy.conf"
```

Comment:
The reason we want to pass this down is for testing.  I agree, I am not sure if skopeo inspect requires it either.  I think it is only used for skopeo copy.  Although --policy is an option for skopeo, rather then just an option for skopeo copy,  @runcom and/or @mtrmac do any of the other skopeo calls use the `--policy` option?